src/pages/Validators/Index.tsx

```diff
@@ -8,14 +11,48 @@ import ValidatorsMap from "./ValidatorsMap";
 
 export default function ValidatorsPage() {
   const [state, _] = useGlobalState();
+  const {config} = useConfig(STAKING_BANNER_CONFIG_NAME);
+  const viewCountCap = config.getValue("viewCount");
+  // Get the user's stable ID
+  const stableID = Statsig.getStableID();
+
+  // Create a key for storing the view count and last visit timestamp in localStorage
+  const viewCountKey = `${stableID}_view_count`;
+  const lastVisitKey = `${stableID}_last_visit`;
+
+  // Get the current view count and last visit timestamp from localStorage
+  const currentViewCount = localStorage.getItem(viewCountKey);
+  const lastVisitTimestamp = localStorage.getItem(lastVisitKey);
+
+  useEffect(() => {
+    if (viewCountCap && Number(currentViewCount) <= Number(viewCountCap)) {
```

Comment:
can `viewCountCap` be null?

Reply:
yes, if `viewCount` is not not set in statsig or we couldn't find the config itself